app/services/article-comments.js

```diff
@@ -2,6 +2,12 @@ import Service, { inject as service } from '@ember/service';
 import { ArticleCommentsFetchError } from '../utils/errors';
 import { track, trackActions } from '../utils/track';
 
+const bannerNotificationTypesMap = {
```

Comment:
The DS notification types are different than the ones in UCP. We could either leave this mapping or unify the types.